examples/blueprint-construct/index.ts

```diff
@@ -149,6 +155,17 @@ export default class BlueprintConstruct {
             new blueprints.AwsBatchAddOn(),
             new blueprints.AwsForFluentBitAddOn(),
             new blueprints.FluxCDAddOn(),
+            // new blueprints.FluxCDAddOn({
```

Comment:
please remove the commented content, however I think it would quite handy for testing if you moved it to the examples file here: https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/examples/examples.ts

That one contains examples all of which deploy to default VPC to avoid address limits. you can run them with `npm run example list|deploy`

Reply:
Done.